### DISCUSSION TITLE
Welcome to pfsense-azure-sentinel Discussions!

### DISCUSSION THREAD

**Discussion by noodlemctwoodle** [noodlemctwoodle/pf-azure-sentinel#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by obsjmartin** [comment]
First off, thanks so much for setting this up! This is invaluable information and I'm sure this project will get more respect once Azure Sentinel is mainstream.

I have a question about the end of the tutorial, specifically the log_type. Where do you find that in the workspace? Do you have to create a log_type for "pfsense_logstash" in the workspace? 
zure_loganalytics {
        customer_id => "<OMS WORKSPACE ID>"
        shared_key => "<CLIENT AUTH KEY>"
        log_type => "<LOG TYPE NAME>"

**Reply by noodlemctwoodle** [reply]
Hello @obsjmartin 

You do not need to configure a custom log source in Azure Sentinel "Advanced settings"

The new custom log will be created automatically by the Azure Log Analytics plugin for Logstash. You should find the pfSense/opnSense table in Azure Sentinel > Logs > Custom Logs, as shown below

![image](https://user-images.githubusercontent.com/15075388/107572180-d29a2e00-6be3-11eb-9270-fcc5bba96933.png)

I've updated the guide to reflect this location 👍

**Reply by noodlemctwoodle** [reply]
I have also given a better example of how to configure these settings in the [guide](https://github.com/noodlemctwoodle/pf-azure-sentinel/blob/main/Logstash-Configuration/README.md#install-and-configure-the-log-analytics-plugin-for-logstash) 

     output {
         azure_loganalytics {
             customer_id => "<WORKSPACE ID>"
             shared_key => "<Primary Key>"
             log_type => "<Name of Log>"
         }
     }
